types/src/block.rs

```diff
@@ -270,7 +270,8 @@ mod tests {
             validator_address,
             timestamp,
             ..
-        } = commit.signatures[0].clone() else {
+        } = commit.signatures[0].clone()
+        else {
```

Comment:
changes in `types` are outside of scope of the PR, but CI fmt was unhappy about it.

Reply:
yeah, the rustfmt got updated with let else formatting at 1.72